app/services/notifiers/rdv_cancelled.rb

```diff
@@ -12,7 +12,7 @@ def notify_user_by_mail(user)
   def notify_user_by_sms(user)
     # Only send sms for excused cancellations by an Agent (not for no-show, not for self-cancellation)
     return unless @author.is_a? Agent
-    return unless @rdv.status.in? %w[excused revoked]
+    return unless @rdv.status.in?(%w[excused revoked]) || @rdv.collectif?
```

Comment:
Aucun SMS n'est envoyé à un usager qui annule sa participation à un RDV Collectif ?

Reply:
pour le moment non, c'est un oubli qu'il y a eu dès la première version des rdv collectifs :(

Reply:
C'est cool du coup de l'avoir ajouté.
Je m'interroge sur l'importance de notifier des usagers qui finalement ne participe plus à un atelier collectif. J'imagine que lorsque c'est elle qui décide de ne plus venir, c'est ok de ne pas avoir de confirmation par SMS (c'est du `nice to have` on va dire) mais lorsque c'est annulé à l'initiative du service, c'est peut-être nécessaire ?

Un ticket à ajouter pour plus tard si ça te conviens ?

Reply:
Je ne suis pas sûr de comprendre quel serait le ticket 🤔  Pour être clair : le correctif qu'on ajoute ici fait que la notification d'annulation sera bien envoyée par sms pour les rdv collectifs

Reply:
Ahah ! :facepalm: j'avais lu de travers. Merci !